src/ign.cc

```diff
@@ -267,3 +272,108 @@ extern "C" SDFORMAT_VISIBLE int cmdGraph(
 
   return 0;
 }
+
+//////////////////////////////////////////////////
+extern "C" SDFORMAT_VISIBLE int cmdInertialStats(
+    const char *_path)
+{
+  if (!sdf::filesystem::exists(_path))
+  {
+    std::cerr << "Error: File [" << _path << "] does not exist.\n";
+    return -1;
+  }
+
+  sdf::Root root;
+  sdf::Errors errors = root.Load(_path);
+  if (!errors.empty())
+  {
+    for (auto &error : errors)
+    {
+      std::cerr << error << std::endl;
+    }
+    return -1;
+  }
+
+  if (root.WorldCount() > 0)
+  {
+    std::cerr << "Error: Expected a model file but received a world file"
+            << std::endl;
+    return -1;
+  }
+
+  const sdf::Model * model = root.Model();
+  if (!model)
+  {
+    std::cerr << "Error: Could not find the model." << std::endl;
+    return -1;
+  }
+
+  ignition::math::Inertiald totalInertial;
+
+  for (uint64_t i = 0; i < model->LinkCount(); i++)
```

Comment:
It just occurred to me that this won't include the inertial properties of links in nested models. Can you add a warning message if `model->ModelCount() > 0`?

Reply:
Done